packaging/package.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+set -e
```

Comment:
u and pipefail flags are also good to have. Also x flag would be good in case there are issues in CI builds

Reply:
u flag cannot be used in this script because the /opt/ros/foxy/setup.bash script has some variables which are not set and we get unbound error. This script is not under our control.